server/src/main/java/com/linecorp/centraldogma/server/storage/project/InternalProjectInitializer.java

```diff
@@ -88,6 +129,7 @@ public void initialize() {
         try {
             initialize0(INTERNAL_PROJECT_DOGMA);
             initializeTokens();
+            initializeSessionKey();
```

Comment:
Couldn't we safely call this method until `Command.createSessionMasterKey()` is deployed?

Reply:
We have to do this in this order:
```
// dogma.json
{
  "encryptionAtRest: {
    "enabled": true,
    "encryptSessionCookie": false
  }
}
```
```
// dogma.json
{
  "encryptionAtRest: {
    "enabled": true,
    "encryptSessionCookie": true
  }
}
```